dash-spv-bench/run.sh

```
@@ -0,0 +1,329 @@
+    done
+  done
+  echo "${peer}"
+}
+
+MODE="$(scn '.mode // "local"')"
+case "${MODE}" in local | testnet | mainnet) ;; *) echo "Error: mode must be 'local', 'testnet' or 'mainnet' (got '${MODE}')" >&2; exit 1 ;; esac
+export BENCH_MODE="${MODE}"
+export CLONE_DIR="${CLONE_DIR:-/nonexistent}"
+
+BENCH_CPUS="$(scn '.cpus // ""')"
+export BENCH_MAX_PEERS="$(scn '.max_peers // ""')"   # only if set; else the ClientConfig default
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)